metadata.json

```diff
@@ -32,6 +32,10 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": ["14.04", "16.04", "17.04", "17.10", "18.04", "18.10"]
+    },
+    {
+      "operatingsystem": "Windows",
```

Comment:
We might want to leave this out for the first release, so we don't announce it as compatible with Windows yet. Comments?

Reply:
Makes sense, or release it as a beta feature? It shouldn't affect linux otherwise.